setup.py

```diff
@@ -45,7 +45,7 @@
     include_package_data=True,
     python_requires=">=3.6",
     install_requires=[
-        'Django>=3.1, <3.2',
+        'Django>=3.2, <4.1',
```

Comment:
I don't agree with increasing this minimum unless there is a new incompatibility with Django 3.1. For the same reasons I wouldn't be in favor of dropping support for Python 2.7 unless new code makes that completely incompatible, despite its obvious age.

Reply:
expanded the django version range